packages/module/src/SystemMessageEntry/SystemMessageEntry.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+
+export interface SystemMessageEntryProps {
+  text: string;
+  href: string;
```

Comment:
What is this `href`?

Reply:
In the ticket, it asked for the ability to add a link.

Reply:
I think we can remove text and href in favor of children ([see my other comment](https://github.com/patternfly/virtual-assistant/pull/9/files#r1550275738))